MaskRCNN_no_batch_convergence/eval.py

```diff
@@ -192,10 +297,12 @@ class EvalCallback(Callback):
 
     _chief_only = False
 
-    def __init__(self, eval_dataset, in_names, out_names, output_dir):
+    def __init__(self, eval_dataset, in_names, out_names, output_dir, batched, batch_size):
         self._eval_dataset = eval_dataset
         self._in_names, self._out_names = in_names, out_names
         self._output_dir = output_dir
+        self.batched = batched
```

Comment:
Do we need both batched and batch_size args? Can't we get rid of the `batched` argument and let `self.batched = batch_size > 1`?

Reply:
Done.